__tests__/pages/api/graphql.test.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * @jest-environment node
```

Comment:
What does it do? I can't found on the doc...

Reply:
I think `jest` tests are by default running in windows environment (to provide browser functionalities). This causes issues for some modules (like sequelize) that relies on core nodejs modules. Setting the env will make this test run in node env. @aortizoj15 could you add a comment?